### DISCUSSION TITLE
Ability to View as Layer Based Editing

### DISCUSSION THREAD

**Discussion by sirinath** [GraphiteEditor/Graphite#1465] [Ideas/Feedback]
While using the node graph add the ability for the user to switch between node based and layer based workflows. 

The layer based view will be translated to the node based view internally.

This way everyone will feel at home and one can switch to the most productive view.

**Comment by Keavon** [comment]
This is already the case and will be further improved going forward. Please look at our [roadmap](https://graphite.rs/features/#roadmap) for more of our plans.

**Reply by sirinath** [reply]
That's great. This was not that clear to me.
